Applications/Spire/Source/Ui/OverlayPanel.cpp

```diff
@@ -131,18 +132,21 @@ void OverlayPanel::keyPressEvent(QKeyEvent* event) {
   QWidget::keyPressEvent(event);
 }
 
+void OverlayPanel::resizeEvent(QResizeEvent* event) {
+  update_mask();
+  QWidget::resizeEvent(event);
+}
+
 void OverlayPanel::position() {
   if(m_positioning == Positioning::PARENT) {
     auto parent_geometry = parentWidget()->rect();
     auto parent_bottom_left = parentWidget()->mapToGlobal(
       parent_geometry.bottomLeft());
     auto screen_geometry = parentWidget()->screen()->availableGeometry();
-    auto panel_size = [&] {
-      if(layout()->contentsMargins() == DROP_SHADOW_MARGINS()) {
-        return size() - QSize(0, DROP_SHADOW_HEIGHT());
-      }
-      return size();
-    }();
+    auto panel_size = size();
+    if(auto win = windowHandle()) {
```

Comment:
In which cases does the OverlayPanel not have a windowHandle?

Reply:
I am not sure. Just in case it may be null.